bears/general/CPDBear.py

```diff
@@ -127,4 +128,5 @@ def run(self, language: str, minimum_tokens: int=20,
                         'that you have more code than you need. Consider'
                         ' refactor your code to remove one of the'
                         ' occurrences. For more information go here:'
-                        'http://tinyurl.com/coala-clone'))
+                        'http://tinyurl.com/coala-clone'),
```

Comment:
to clarify: http://bit.ly/line-spots+ sends you to a page that shows you where http://bit.ly/line-spots leads to. (The plus at the end does that)